.github/workflows/wheels.yaml

```diff
@@ -1,28 +1,30 @@
 name: Wheels
-'on':
+"on":
   pull_request: ~
   push:
     branches:
-      - 'master'
+      - "master"
     tags:
-      - '[0-9]+.[0-9]+.[0-9]+*'
-      - 'v[0-9]+.[0-9]+.[0-9]+*'
+      - "[0-9]+.[0-9]+.[0-9]+*"
+      - "v[0-9]+.[0-9]+.[0-9]+*"
 jobs:
   build_wheels:
     name: Build wheels on ${{ matrix.os.name }}
     runs-on: ${{ matrix.os.name }}
     strategy:
       matrix:
         os:
-          - {name: "ubuntu-20.04", needs_qemu: true}
-          - {name: "windows-2019", needs_qemu: false}
-          - {name: "macOS-11", needs_qemu: false}
+          - { name: "ubuntu-20.04", needs_qemu: true }
+          - { name: "windows-2019", needs_qemu: false }
+          - { name: "macOS-11", needs_qemu: false }
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
           submodules: recursive
       - uses: actions/setup-python@v3
+        with:
+          python-version: ">=3 <3.12"
```

Comment:
Had to use <3.12 because distutils was removed from python 3.12
https://docs.python.org/3/whatsnew/3.12.html#:~:text=Python%203.12%20is%20the%20latest,removed%20from%20the%20standard%20library.